### PR TITLE
Removed reporting of MachineID to Google Analytics.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-858" title="DX-858" target="_blank">StoryDX-858</a>  Fully remove machineID dimension
  </td></table>
  <br />
 

It was replaced by DeviceID.

### CODE REVIEW

**Review by Naatan — CHANGES_REQUESTED** [review]
We should be able to drop the machineid package.

**Comment by mitchell-as** [comment]
I tried, but we are using it in other places unrelated to analytics, for example https://github.com/ActiveState/cli/blob/9fe4c6c6143a1d4b21e3c9c399545362bfa0815c/pkg/platform/model/vcs.go#L386 and https://github.com/ActiveState/cli/blob/9fe4c6c6143a1d4b21e3c9c399545362bfa0815c/pkg/platform/authentication/auth.go#L364

File: internal/rollbar/rollbar.go [hunk 1]
```diff
@@ -93,7 +92,7 @@ func UpdateRollbarPerson(userID, username, email string) {
 	}
 
 	custom["UserID"] = userID
-	custom["MachineID"] = machineid.UniqID()
+	custom["MachineID"] = uniqid.Text()
```
Comment (Naatan) [inline internal/rollbar/rollbar.go:95]:
```suggestion
	custom["DeviceID"] = uniqid.Text()
```

File: pkg/platform/model/vcs.go [hunk 2]
```diff
@@ -383,7 +382,7 @@ func AddChangeset(parentCommitID strfmt.UUID, commitMessage string, changeset Ch
 		Changeset:      changeset,
 		Message:        commitMessage,
 		ParentCommitID: parentCommitID,
-		AnonID:         machineid.UniqID(),
+		AnonID:         uniqid.Text(),
```
Comment (Naatan) [inline pkg/platform/model/vcs.go:385]:
```suggestion
```

**Review by Naatan — CHANGES_REQUESTED** [review]

**Review by Naatan — APPROVED** [review]

**Pull request merged**